lib/livebook/runtime/evaluator/default_formatter.ex

```diff
@@ -128,4 +128,7 @@ defmodule Livebook.Runtime.Evaluator.DefaultFormatter do
   defp error_color(string) do
     IO.ANSI.format([:red, string], true)
   end
+
+  defp error_type(%System.EnvError{env: "LB_" <> secret}), do: {:missing_secret, secret}
```

Comment:
So in the end we can make it `:unavailable_env` and match on prefix on livebook side, no?

Reply:
I would prefer to keep it as is because otherwise we will need to match on it twice. One for the env, another for the secret. It just looks like more work. :D

Reply:
I'm just saying that we generalized `put_secrets` into `put_envs`, so we could also avoid the secret output (and runtime knowing about the `LB_` prefix). If you think this doesn't matter, it's fine by me though :)

Reply:
To be clear, what I suggest only differs in that we move `"LB_" <>` to the LV side.

Reply:
I am fine either way :)

Reply:
The runtime (smart cells) will have to know that secrets are env vars with "LB_" suffix. So the special handling of "LB_" is "common knowledge" in the runtime.

Reply:
I guess then one of the main reasons to avoid `add_secrets` in the runtime is mostly because `add_system_env` is more general purpose?

Reply:
Fair point! I guess the general purpose argument could go for the output, but at this point I'm fine with whichever, especially that we this we can change anytime :)